projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-wpcom-online-subscription-service.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * This subscription service is used when a subscriber is offline and a token is not available.
+ * This subscription service will be used when rendering content in email and reader on WPCOM only.
+ * When content is being rendered, the current user and site are set.
+ * This allows us to lookup a users subscriptions and determine if the
+ * offline visitor can view content that has been deemed "Premium content".
+ *
+ * @package Automattic\Jetpack\Extensions\Premium_Content
+ */
+
+namespace Automattic\Jetpack\Extensions\Premium_Content\Subscription_Service;
+
+/**
+ * Class WPCOM_Offline_Subscription_Service
```

Comment:
This is a lie. @millerf I don't follow the rename of `WPCOM_Offline_Subscription_Service` to `WPCOM_Online_Subscription_Service`

Reply:
Have a look at the available() function